src/zks_wallet/wallet.rs

```diff
@@ -1010,6 +1070,8 @@ mod zks_signer_tests {
 
         println!("L2 Transaction hash: {:?}", tx_receipt.transaction_hash);
 
+        tokio::time::sleep(Duration::from_secs(10)).await;
```

Comment:
you should be able to take this code to avoid the sleep: https://github.com/lambdaclass/zksync-web3-rs/pull/59

Reply:
Great, I will totally use that.